ci_framework/roles/build_containers/tasks/main.yml

```diff
@@ -35,13 +35,18 @@
 
 - name: Run container_build.sh script
   become: true
+  when: not cifmw_build_containers_run_hotfix | bool
   args:
     chdir: "{{ cifmw_build_containers_basedir }}/artifacts"
   ansible.builtin.shell: |
     set -o pipefail && bash build_containers.sh 2>&1 {{ cifmw_build_containers_timestamper_cmd }} > {{ cifmw_build_containers_basedir }}/logs/build.log
   environment:
     PATH: /usr/local/bin:{{ ansible_env.PATH }}
 
+- name: Include tasks from hotfix.yml
+  when: cifmw_build_containers_run_hotfix | bool
+  ansible.builtin.include_tasks: hotfix.yml
```

Comment:
if it's not something we call multiple time, please consider using `ansible.builtin.import_tasks` instead. But only if it's called one once (i.e. no loop calling the whole role multiple times)

Reply:
@cjeanner it is called once in the whole playbook, so updating this with import_tasks.